sql/catalyst/src/main/java/org/apache/spark/sql/catalog/v2/TableCapability.java

```diff
@@ -47,23 +47,23 @@
    * <p>
    * Truncating a table removes all existing rows.
    * <p>
-   * See {@link org.apache.spark.sql.sources.v2.writer.SupportsTruncate}.
```

Comment:
We should consider moving the v2 source API into the catalyst module as well. That would fix these links, but it a much larger issue. I'll plan on bringing it up at the next v2 sync.